custom/icds_reports/sqldata/exports/children.py

```diff
@@ -54,241 +54,703 @@ def get_columns_by_loc_level(self):
     @property
     def columns(self):
         columns = self.get_columns_by_loc_level
-        agg_columns = [
-            AggregateColumn(
-                'Weighing efficiency (in month)',
-                percent,
-                [
+
+
+        if self.beta:
```

Comment:
I don't know what's changing in this function because so much of it has moved. Removing whitespace changes doesn't help much either. It would be easier to review if it followed a pattern such as:

```python
agg_columns = [...]
if self.beta:
    agg_columns.insert(0, first_column) 
    agg_columns.insert(1, second_column)
    agg_columns.extend(columns_at_end)
```

That should make it much more clear what this feature is doing and what the difference between these features are

Reply:
sure. I will do this in 1 hour